static/js/marker.js

```diff
@@ -39,7 +39,15 @@ var MarkerView = Backbone.View.extend({
 
 		this.$el.width(400);
 		this.$el.find(".title").text(TYPES_MAP[this.model.get("title")]);
-		this.$el.find(".description").text(this.model.get("description"));
+		this.$el.find(".roadType").text(this.model.get("roadType"));
```

Comment:
1. Prefix the value by the field name (as in תאריך below).
2. Get the value as integer and localize on the client, to avoid saving text in the DB and sending it to the client.